src/data/news/coverage.yml

```diff
@@ -341,6 +383,7 @@
   Params:
     sortDate: "2019-07-15"
     icon: "broadcasts.svg"
+    outlet: "DecredTV"
```

Comment:
I'm not sure if I like using the name DecredTV here, feels weird because I've never seen it anywhere before.

Are we trying to establish it as a new brand?

Reply:
I used channel name for `outlet` for all content on YouTube (or more generally, not hosted on its own website). This often matches the established brand name e.g. "Digital Cash Network", "Off The Chain Podcast", "POV Crypto Podcast", etc.

This was a bit tricky. I didn't want to use "YouTube" because I see it as a platform for many outlets (i.e. lower level) and not an "outlet" on its own. A generic "Decred" does not feel right either.

Now that I'm saying it, "Decred in Depth Podcast" looks better since it's an established series title in the community. Changed to use that, but I can change to anything else to not block on this decision and we can sort it out later.

Reply:
Agreed, DiDP is better